src/app/forms/custom-sections/plates/plates.component.html

```diff
@@ -0,0 +1,35 @@
+<form [formGroup]="form">
+  <dl class="govuk-summary-list">
+    <ng-container *ngIf="hasPlates()">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Plate serial number</dt>
+        <dd id="test-cor" class="govuk-summary-list__value">{{ plateSerialNumber ?? '-' }}</dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Plate issue date</dt>
+        <dd id="test-cor" class="govuk-summary-list__value">{{ plateIssueDate ?? '-' }}</dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Plate issuer</dt>
+        <dd id="test-cor" class="govuk-summary-list__value">{{ plateIssuer ?? '-' }}</dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Plate:</dt>
+        <dd id="test-certificate" class="govuk-summary-list__value">
+          <button id="test-certificate-link" class="link" (click)="download()">{{ 'plate_' + plateSerialNumber + '.pdf' }}</button>
+        </dd>
+      </div>
+    </ng-container>
+    <ng-container *ngIf="!hasPlates()">
+      <div>
+        <dt class="govuk-summary-list__key">This vehicle does not have a plate.</dt>
+      </div>
+    </ng-container>
+    <div class="govuk-grid-row">
+      <br />
+      <div class="govuk-grid-column-full">
+        <app-button (clicked)="generatePlate()" *appRoleRequired="roles.TechRecordAmend">Generate plate</app-button>
```

Comment:
```suggestion
        <app-button *appRoleRequired="roles.TechRecordAmend" routerLink="generate-plate">Generate plate</app-button>
```
And you can remove `generatePlate()`.

Reply:
Tried this change - didn't navigate to generate-plate

Reply:
Apologies, my bad. If you want to use `routerLink` you also need to specify `type="link"`. So, it would look like this:
```suggestion
        <app-button *appRoleRequired="roles.TechRecordAmend" type="link" routerLink="generate-plate">Generate plate</app-button>
```